keyboards/quokka/rules.mk

```diff
@@ -0,0 +1,14 @@
+BOARD = GENERIC_RP_RP2040
+CONVERT_TO = kb2040
```

Comment:
This shouldn't really be using a convert at the kb level.  Either it should target pro micro, or it should be RP2040 completely (and GP## pins for the defines)

Reply:
@drashna I tried targeting RP2040 and using `GP##` pins, but setting `BOOTLOADER = rp2040` in `rules.mk` won't build, I get the error `invalid bootloader specified. Please set an appropriate bootloader in your rules.mk or info.json`. I looked through the docs but couldn't find anything that resolved the issue.

Reply:
Figured it out, a few more tweaks to make still

Reply:
Finished, targeting RP2040 completely